analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -106,6 +106,7 @@ Rscript 03-Calc-zscore-annotate.R --standardFusionCalls "${scratch_path}/standar
                                   --normalExpressionMatrix $normal_expression_file \
                                   --outputfile "${scratch_path}/standardFusionStrandedExp_QC_expression"
 
+if [ "$RUN_FOR_SUBTYPING" == 0 ]; then
```

Comment:
I wasn't expecting this add. Is it something that was meant to be added previously but didn't get in?

Reply:
hmm this script is not part of fusion_summary module but would have been part of #865 which is already merged now. I'll revert these analyses/fusion_filtering/* files to master.

Reply:
ok reverted those files

Reply:
Just double checking: These changes are all intentional: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/866/commits/ca25c8dc172a5c1669e791d099146a5d7c23c496

Reply:
Looking back at your description, I think yes. I think we are all set.

Reply:
correct, thanks for double checking these are intentional to update to the module in master . 

I must have missed merging the new changes from fusion_filtering-base to this branch which left these older file in this branch which was staggered from fusion_filtering_base branch.